packages/terra-folder-tree/tests/wdio/folder-tree-spec.js

```diff
@@ -1,19 +1,34 @@
-Terra.describeViewports('FolderTree', ['medium', 'large'], () => {
+Terra.describeViewports('FolderTree', ['small', 'medium'], () => {
```

Comment:
Do we need both screen sizes for this?

Reply:
I'm not sure which would be more appropriate here. Maybe just medium?

Reply:
Seems like you're testing basic functionality that doesn't correspond to screen size. I would suggest just doing medium or large so we don't have redundant tests.

If you want to test layout at various widths, then we can have specific tests for those screen sizes.